testing/simapp/test_helpers.go

```diff
@@ -31,13 +25,12 @@ import (
 	dbm "github.com/tendermint/tm-db"
 
 	"github.com/cosmos/ibc-go/v6/testing/mock"
-	"github.com/cosmos/ibc-go/v6/testing/simapp/helpers"
 )
 
 // DefaultConsensusParams defines the default Tendermint consensus params used in
 // SimApp testing.
-var DefaultConsensusParams = &abci.ConsensusParams{
-	Block: &abci.BlockParams{
+var DefaultConsensusParams = &tmproto.ConsensusParams{
```

Comment:
Is it still required to pass `ConsensusParams` to `abci.RequestInitChain` now that consensus params are managed from their own sdk module?

This is used in our testing package [here](https://github.com/cosmos/ibc-go/blob/main/testing/simapp/test_helpers.go#L102) and [here](https://github.com/cosmos/ibc-go/blob/main/testing/simapp/test_helpers.go#L203)

Reply:
cc. @tac0turtle

Reply:
Yes I believe so. The consensus params are used to modify tendermint consensus parameters. Moving consensus params to their own SDK module was changing how SDK stores the values but not modifying how SDK tells tendermint of what the values are

Reply:
Cool, that makes sense to me!